lib/discourse.rb

```diff
@@ -1141,6 +1141,7 @@ def self.preload_rails!
         Discourse.git_version
         Discourse.git_branch
         Discourse.full_version
+        Discourse.plugins.each { |p| p.commit_url }
```

Comment:
What about `commit_hash`? That is also making a system call as well.

Reply:
Now, the `commit_url` method will call the `commit_hash`